troposphere/static/js/actions/tag/create.js

```diff
@@ -16,7 +16,8 @@ export default {
 
         var tag = new Tag({
             name: name,
-            description: description
+            description: description,
+            allow_access: !(/featured/i).test(name),
```

Comment:
This treats "Featured" as a forbidden word. 
So: "Featured Images" is also not allowed.  

We could be more strict:
```
(/^featured$/i).test("Featuredd")
```